cleaner.go

```diff
@@ -85,6 +88,11 @@ func NewCleaner(kubeConfigContent []byte) (*Cleaner, error) {
 	}, nil
 }
 
+func isCRDMissing(err error) bool {
```

Comment:
I think you can use https://github.com/kubernetes/apimachinery/blob/master/pkg/api/meta/errors.go#L111

Reply:
that is a good point, swapped in https://github.com/kyma-incubator/sc-removal/pull/8/commits/746568f8ee83ba199d3b30e202c75d20da23e6a9